src/bot/mod.rs

```diff
@@ -100,15 +100,48 @@ impl Bot {
         Self::with_client(&get_env(TELOXIDE_TOKEN), client)
     }
 
-    /// Returns currently used token
+    /// Sets custom api url.
+    ///
+    /// For example you can run your own [Telegram bot API server][tbas] and set
+    /// it's url using this method.
+    ///
+    /// [tbas]: https://github.com/tdlib/telegram-bot-api
+    ///
+    /// ## Examples
+    ///
+    /// ```
+    /// use teloxide_core::{
+    ///     requests::{Request, Requester},
+    ///     Bot,
+    /// };
+    ///
+    /// # async {
+    /// // Setup bot
+    /// let url = reqwest::Url::parse("https://localhost/tbas").unwrap();
+    /// let bot = Bot::new("TOKEN").set_api_url(url);
+    /// // All methods will use "https://localhost/tbas" as an api url
+    /// bot.get_me().send().await
+    /// # };
+    /// ```
```

Comment:
```suggestion
    /// Sets a custom API URL.
    ///
    /// For example, you can run your own [Telegram bot API server][tbas] and set
    /// its URL using this method.
    ///
    /// [tbas]: https://github.com/tdlib/telegram-bot-api
    ///
    /// ## Examples
    ///
    /// ```
    /// use teloxide_core::{
    ///     requests::{Request, Requester},
    ///     Bot,
    /// };
    ///
    /// # async {
    /// let url = reqwest::Url::parse("https://localhost/tbas").unwrap();
    /// let bot = Bot::new("TOKEN").set_api_url(url);
    /// // From now all methods will use "https://localhost/tbas" as an API URL.
    /// bot.get_me().send().await
    /// # };
    /// ```
```